tests/subsys/logging/log_list/README.txt

```diff
@@ -0,0 +1,44 @@
+Title: Log list
+
+Description:
+
+This test verifies that the log list operates as expected.
+
+--------------------------------------------------------------------------------
+
+Building and Running Project:
+
+This project outputs to the console.  It can be built and executed
+on QEMU as follows:
+
+    make run
```

Comment:
This is out of date, there should be CMake here. Also please use ` .. zephyr-app-commands::` for this